### DISCUSSION TITLE
[Question]<davinci with e3d values>

### DISCUSSION THREAD

**Discussion by Honzsolo707** [luc-github/Repetier-Firmware-0.92#369] [Q&A]
What is your question ?
so i got everything flashed, with new e3d installed, chose option 7 on thermistor and it gives me decouple no matter what. change back to option 5 and its fine and reads right. my question would be is the chart going to be somewhat close on the temps? it appears to be printing fine i just dont see how everyone elses worked on option 7 and mine however did not.

**Comment by luc-github** [comment]
the answer is in FAQ https://github.com/luc-github/Repetier-Firmware-4-Davinci/issues/104

eventually try https://github.com/luc-github/Repetier-Firmware-4-Davinci/issues/207

**Comment by luc-github** [comment]
Moving topic to discussion panel
